testing/asserts.ts

```diff
@@ -223,6 +223,13 @@ export function assert(expr: unknown, msg = ""): asserts expr {
   }
 }
 
+/** Make an assertion, error will be thrown if `expr` have truthy value. */
+export function assertFalse(expr: unknown, msg = ""): asserts expr is false {
```

Comment:
```suggestion
export function assertNot(expr: unknown, msg = ""): asserts not expr {
```

This matches other assertions like `assertNotEquals`.

Additionally, `assertFalse` is misleading as this implies only the literal value `false` will pass this assertion when in actual fact this is asserting whether the expression is truthy.

Reply:
That syntax is not valid TypeScript, but you are right, is misleading...

I'm not sure what we could do, any ideas @kt3k  @bartlomieju @bcheidemann ?

Reply:
I looked into this `assert <<cond>>` syntax a bit because I didn't know much about it. I think we could do something like this:

```ts
type Falsy = false | null | undefined | "" | 0;
export function assertNot(expr: unknown, msg = ""): asserts expr is Falsy {
  // --snip--
}
```

The type inference for this is actually pretty smart:

![image](https://user-images.githubusercontent.com/56122437/166100547-4fece0eb-4710-45ce-b44f-9971a9f3689b.png)

![image](https://user-images.githubusercontent.com/56122437/166100522-49b386e8-f68c-40ca-a872-54d5b41c4b35.png)